CorsixTH/Lua/app.lua

```diff
@@ -1671,14 +1671,19 @@ function App:afterLoad()
 end
 
 function App:checkForUpdates()
+  local function extractVersionNumericals(version)
+    local _, _, major, minor, rev = string.find(version, "(%d+).(%d+).(%d+)") -- luacheck: ignore 211
```

Comment:
Apparently string.find outputs 5 arguments? Lua documentation says we should use `_` for unwanted args, even though luacheck doesn't like it.

Reply:
That pattern does according to https://www.lua.org/manual/5.3/manual.html#6.4 

Don't know what luacheck is saying, but perhaps it barks at you trying assign 2 values to the same `_` variable in one assignment? That could be fixed by making the names different, eg `_start, _end, major, minor, rev`